src/components/RnR/RnR_sort_list.jsx

```diff
@@ -11,27 +11,26 @@ class Sort extends Component {
   constructor(props) {
     super(props);
     this.state = this.props;
-
-    this.getAllReviews = () => {
-      const { productId } = this.state;
-      const listData = fetch(`http://18.217.220.129/reviews/${productId}/list`)
-        .then((response) => {
-          if (response.status !== 200) { console.log('problem'); }
-          return response.json();
-        });
-      const metaData = fetch(`http://18.217.220.129/reviews/${productId}/meta`)
-        .then((response) => {
-          if (response.status !== 200) { console.log('problem'); }
-          return response.json();
-        });
-      Promise.all([listData, metaData]).then((info) => { console.log(info); });
-    };
   }
 
   componentDidMount() {
-    this.getAllReviews();
+    const { productId, dispatch } = this.state;
```

Comment:
I would be pulling this from props and do away with state, unless there's an underlying issue with that that I'm not seeing.

Reply:
Or did this have something to do with that setTimeout you slacked me about?